FSQCellManifest/FSQCellManifest.m

```diff
@@ -1552,12 +1552,40 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         NSString *identifier = record.reuseIdentifier ?: NSStringFromClass(record.cellClass);
         
         if (!record || !identifier || !record.cellClass) {
+            NSString *delegateClassName = NSStringFromClass([self.delegate class]) ?: @"";
+            NSString *indexPathString = [NSString stringWithFormat:@"indexPath.section: %@, indexPath.row: %@",
+                                         @(indexPath.section),
+                                         @(indexPath.row)];
+            NSString *tablePathString = [NSString stringWithFormat:@"tableView.numberOfSections: %@, [tableView numberOfRowsInSection:indexPath.section]: %@",
+                                         @(tableView.numberOfSections),
+                                         @([tableView numberOfRowsInSection:indexPath.section])];
+
+            NSString *badOrNilObject = @"";
+            if (!record) {
+                badOrNilObject = @"FSQCellRecord";
+            }
+            else if (!identifier) {
```

Comment:
instead of an else should we unconditionally check these as well and just append them to the string in the chance that all 3 are nil